REST-API-ADVANCED/advanced/src/main/java/com/epam/esm/security/dto/AuthUserDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.esm.security.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class AuthUserDTO {
+    private String username;
```

Comment:
I don't like the `DTO` naming. DTO is type of model, but shouldn't often be used for naming. It is request I believe, so rename it accordingly

Reply:
Renamed all dto as model